styles/index.less

```diff
@@ -222,6 +222,11 @@ html.metrics-compact { @import "metrics-compact"; }
                 var(--item-hover-bg) var(--active-marker-bw));
         }
     }
+
+    &.discarded {
+        opacity: 0.75;
```

Comment:
This also de-saturates the selection and over backgrounds. You might want to consider applying the opacity/saturation only to the `.folder-name` and icon (`.action.select`).

<img width="203" alt="Screenshot 2022-02-27 160424" src="https://user-images.githubusercontent.com/1203893/155905817-81339eca-3346-4cc3-a370-907b61c80fda.png">

Reply:
Good point. Unfortunately, styles can't really apply to `<img>` elements. Something about them being replaced, I think. I changed the style so it only applies the unloaded effect if the tab isn't selected. Does that work for you?

Reply:
Hm, I haven't had a problem with it--for example, `opacity` is used on the icons for the deleted-items page:

https://github.com/josh-berry/tab-stash/blob/12ae9149ad83e0306e32a2220406964d780886b7/styles/index.less#L247

Reply:
My mistake, didn't realize the difference between `.x .y` and `.x.y` in CSS selectors. I'm not really a frontend guy!

This has now been changed to only apply to the text and icon as you suggested.

Reply:
Thanks, looks good!